ci/ray_ci/container.py

```diff
@@ -63,7 +65,7 @@ def docker_login() -> None:
             "--password-stdin",
             DOCKER_ECR,
         ],
-        stdin=password,
+        stdin=p.stdout,
```

Comment:
instead of using the piping, you want to use something like this:

```
with tempfile.TemporaryFile() as f:
    f.write(password)
    f.flush()
    f.seek(0)

    subprocess.check_call(
        [ "your", "command, "here" ],
        stdin=f,
        stdout=sys.stdout,
        stderr=sys.stderr,
    )
```